packages/dependency-extraction-webpack-plugin/lib/types.d.ts

```diff
@@ -13,7 +13,7 @@ declare interface DependencyExtractionWebpackPluginOptions {
 	outputFormat?: 'php' | 'json';
 	outputFilename?: string | Function;
 	requestToExternal?: ( request: string ) => string | string[] | undefined;
-	requestToExternalModule?: ( request: string ) => string | undefined;
+	requestToExternalModule?: ( request: string ) => string | true | undefined;
```

Comment:
A `string | boolean` type makes more sense. `false` is a better value than `undefined`. And lets you write straightforward code like:
```js
function defaultRequestToExternalModule( request ) {
  return request === '@wordpress/interactivity';
}
```